src/components/tasks/card/index.tsx

```diff
@@ -520,7 +520,7 @@ const Card: FC<CardProps> = ({
                         taskId={cardDetails.id}
                     >
                         <span
-                            className={classNames.cardTitle}
+                            className={classNames.cardTitleText}
```

Comment:
Can we remove the CSS for the `cardTitle` class if it's not being used?

Reply:
@sahsisunny, `cardTitle` class is being used on the parent component `ConditionalLinkWrapper` over the `span` title element.

Reply:
Okay